README.md

```diff
@@ -1,2 +1,168 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Weather Today</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Weather Today\] ](#-Weather-Today-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Usage](#usage-1)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [Weather_Today] <a name="about-project"></a>
+
+**[Weather_Today]** is the React capstone project and it is a mobile web application that allows users to check a list of metrics (numeric values) for the weather in different cities around the world. The application is built using React and Redux. The application retrieves the weather data from the [OpenWeather](https://openweathermap.org/api) API. Users can filter the list of countries by region and select a country and a city to see the weather details.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technology</summary>
+  <ul>
+    <li>React</li>
+    <li>Redux</li>
+    <li>Redux Toolkit</li>
+    <li>React Router Dom v6</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tools</summary>
+  <ul>
+    <li>VS Code</li>
+    <li>GIT</li>
+    <li>GITHUB</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+This project is a simple weather metrics app. The main features are:
+
+The app consists of three pages:
+
+- **The Home page displays a list of cities and the weather there.**
+- **The Details page displays the weather details for the selected city.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[N/A]
```

Comment:
- [ ] Kindly consider deploying the application using Render, Netlify or GitHub Pages.